client/my-sites/upgrades/checkout-thank-you/header/index.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const CheckoutThankYouHeader = React.createClass( {
+	propTypes: {
+		isDataLoaded: React.PropTypes.bool.isRequired,
+		isFreeTrial: React.PropTypes.bool.isRequired,
+		productName: React.PropTypes.string.isRequired
+	},
+
+	renderHeading() {
+		let heading = this.translate( 'Thank you for your purchase!' );
+
+		if ( ! this.props.isDataLoaded ) {
+			heading = this.translate( 'Loading…' );
+		} else if ( this.props.isFreeTrial ) {
+			heading = this.translate( 'Your 14 day free trial starts today!' );
+		}
+
+		return (
+			<h1 className="checkout-thank-you-header__heading">
+				{ heading }
+			</h1>
+		);
+	},
+
+	renderText: function() {
```

Comment:
You use the short function syntax elsewhere in this file, so you should probably do the same here.